### PR TITLE
Move hardcode into environment variables and use grammy rewrite Telegram bot 

### PR DESCRIPTION
**Pull request by ChestnutLUO**
Not sure about moving hardcode(gateway_name,model) into environment variables and renaming wrangler.toml to wrangler-template.toml to allow user configuration, which seems unreasonable, so I break it down into separate commits.

## Summary by CodeRabbit

- **New Features**
	- Transitioned the Telegram bot implementation to the `grammy` framework, enhancing command handling and message processing.
	- Introduced a new configuration template for setting up a Cloudflare Worker, optimizing performance and resource binding.

- **Bug Fixes**
	- Improved error handling for commands, ensuring better user feedback and input validation.

- **Documentation**
	- Added comprehensive configuration details for the Cloudflare Worker setup, including variable bindings and trigger schedules.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -103,171 +99,179 @@ export default {
...
-								userName, // not interested in user id
-								content,
-								messageId,
-								groupName
-							)
-							.run();
-						return new Response('ok');
+		const bot = new Bot(env.SECRET_TELEGRAM_API_TOKEN);
+		bot.command('status', async (ctx) => {
+            await ctx.reply('我家还蛮大的',{reply_parameters: {message_id: ctx.msg.message_id},});
```
[the remark this answers is not in the document]
Comment (ChestnutLUO) [reply src/index.ts:104]: I'm not sure about it. So I just put what document link here.
https://grammy.dev/guide/basics#sending-messages-with-reply

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@codebam/cf-workers-telegram-bot": "^7.27.0",
```
Comment (asukaminato0721) [inline package.json:20]: we can uninstall it now.

File: src/index.ts [hunk 3]
```diff
@@ -103,171 +103,179 @@ export default {
...
+            return;
+        }
+
+        const { results } = await env.DB.prepare(`
+            SELECT * FROM Messages
+            WHERE groupId=? AND content GLOB ?
+            ORDER BY timeStamp ASC
+            LIMIT 2000`)
+            .bind(groupId, `*${messageText.split(" ")[1]}*`)
+            .all();
```
[the remark this answers is not in the document]
Comment (asukaminato0721) [reply src/index.ts:126]: no, it should support cjk

File: src/index.ts [hunk 4]
```diff
@@ -103,171 +99,179 @@ export default {
...
+	            .run();
+	    }
+
+	    // Handle photo messages
+	    if (msg.photo) {
+				const photo = msg.photo[msg.photo.length - 1];
+				const file = await ctx.api.getFile(photo.file_id);
+				const response = await fetch(`https://api.telegram.org/file/bot${env.SECRET_TELEGRAM_API_TOKEN}/${file.file_path}`);
+        const arrayBuffer = await response.arrayBuffer();
+
```
[the remark this answers is not in the document]
Comment (asukaminato0721) [reply src/index.ts:262]: no, it should not reply in this situation.

**Pull request merged**